test_general/api/test_app.py

```diff
@@ -601,7 +601,7 @@ def test_parallel_mode_preserves_uniqueness_of_hashes_when_assembling_pages_spli
     response = client.post(
         MAIN_API_ROUTE,
         files=[("files", (str(test_file), open(test_file, "rb"), "application/pdf"))],
-        data={},
+        data={"chunking_strategy": "by_title", "new_after_n_chars": 500},
```

Comment:
@awalker4 - I get `<Response [422 Unprocessable Entity]>.status_code` when I use `data={}`, and the associated error message is below. When you explicitly set the params it complains about, the test works fine. Will check and see if any dependency bumps seem like they'd cause this.

```
ERROR    unstructured_api:app.py:44 [{'type': 'literal_error', 'loc': ['body', 'chunking_strategy'], 'msg': "Input should be 'by_title'", 'input': '', 'ctx': {'expected': "'by_title'"}}, {'type': 'int_parsing', 'lo
c': ['body', 'new_after_n_chars'], 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': ''}]
```

Reply:
Nothing jumps out unless it was the `fastapi` version bump. I'll give that a try  quick to test.

Reply:
Confirmed, the `fastapi` bump is what caused the failure. Works with `fastapi==0.112.x` and `fastapi==0.113.x` but not `fastapi==0.114.x`.

Reply:
Pinning to `fastapi==0.113.0` for number to preserve behavior, but we should circle back on how to safely upgrade `fastapi`.